xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1132,6 +1132,13 @@ struct AppModel: ModelProtocol {
     ) -> Update<AppModel> {
         var model = state
         model.isFirstRunComplete = isComplete
+        if isComplete {
+            return update(
+                state: model,
+                action: .fetchNicknameFromProfile,
+                environment: environment
+            )
+        }
```

Comment:
I've realised that we'd benefit from something like #637 here. 

We want to notify whenever the nickname is changed and re-fetch it from the profile memo to keep `app.state.nickname` properly in-sync.